lib/scarpe/para.rb

```diff
@@ -7,11 +7,16 @@ def initialize(app, text)
     end
 
     def render
-      "<p id=#{object_id}>#{text.join}</p>"
+      "<p id=#{object_id}>#{render_text}</p>"
```

Comment:
Should we move this to `HTML.render` now?

Reply:
Rebasing took care of this for me :D